Add in Tutorial on Replicating and Browsing

ebarry opened the pull request:
hi folks, i did another pass to get closer.

ebarry committed:
Update README.md

updated screenshot, simplified language, deleted: 

## Key Concepts

* Public Record
* Data Together Nodes
* Distributed Data Stewardship

ebarry committed:
Update README.md

ebarry committed:
Update README.md

ebarry committed:
Update README.md

ebarry committed:
Update README.md

ebarry committed:
Create README.md

mirror the structure of the previous tutorial and make space for rapidly filling in by those who know the detailed steps

ebarry committed:
Create delete-me.md

ebarry committed:
Add files via upload

ebarry committed:
Delete delete-me.md

ebarry committed:
Update README.md

ebarry committed:
Add files via upload

ebarry committed:
Update README.md

ebarry committed:
Update README.md

ebarry committed:
Update README.md

ebarry committed:
Create README.md

dcwalk commented:
This looks great! I'm just going to add in the Sidebar/summary stuff, hopefully this afternoon we can get some content in and rebuild

b5 commented:
@ebarry is working on scaffolding for the first & second lessons, both visible in the branch. 

@b5 suggested using a "happy path" version that serves fewer edge cases, including reference to the "deeper" distributed web primer that includes more details. 

@ebarry needs help with a second lesson on getting the hash of data. one of us should connect to make sure all concepts are clear.

dcwalk committed:
add table of contents for new tutorial

b5 commented on add-dataset/README.md:
```diff
@@ -62,11 +56,13 @@ Adding, harvesting, and storing are all verbs used in Data Together to describe
 
+* Public Record means [...]
+* Data Together Nodes means [...]
+* Distributed Data Stewardship means [...]
```
The definition of these may come up in conversation at the 4S event, but generally we should put time into defining these!

ebarry committed:
Update README.md

b5 commented:
Quick potential definitions for tutorial:

**The Public Record**: a shared record of the information that has been published by the plethora of entities on the web
**Data Together Nodes means**: Let's leave this definition off for now, as Data Together nodes don't yet exist
**Distributed Data Stewardship** is using distributed technologies to secure & provide access to data that you care about.

ebarry committed:
Add files via upload

ebarry committed:
Update README.md

added definitions, removed un-needed step.

ebarry committed:
Update README.md

ebarry committed:
Update README.md

ebarry committed:
Update README.md

ebarry committed:
Update README.md

ebarry committed:
Add files via upload

ebarry committed:
Delete Screen Shot 2017-08-25 at 7.03.21 PM.png

ebarry committed:
Add files via upload

ebarry committed:
Update README.md

added screenshots of archivers.co steps

ebarry committed:
Update README.md

ebarry committed:
Add files via upload

ebarry committed:
Update README.md

ebarry committed:
Update README.md

ebarry committed:
Update README.md

ebarry committed:
Update README.md

ebarry committed:
Update README.md

ebarry committed:
Update README.md

ebarry committed:
Update README.md

ebarry committed:
Update README.md

ebarry committed:
Update README.md

ebarry committed:
Create delete-me.md

ebarry committed:
Add files via upload

ebarry committed:
Delete delete-me.md

ebarry committed:
Update README.md

ebarry committed:
Update README.md

ebarry committed:
Update README.md

ebarry committed:
Update README.md

ebarry committed:
Update README.md

mhucka committed:
Removed links to nonexistent sections

I think the two subsections came from an earlier draft of the gitbook,
but they didn't resolve to anything now and clicking them did nothing
(at least in my environment).  I removed them entirely for now.

mhucka committed:
Changed image links to be relative

The links originally pointed to blobs on GitHub, but didn't resolve to
images in my local environment for some reason. Changing them to
relative links seems to make sense for the eventually installation
anyway, so I went ahead and did that, to feed two birds with one scone.

mhucka committed:
Edit -- very minor punctuation and formatting changes

mhucka committed:
Replace {hash} with X in explanations

This is maybe controversial, but I find that adding any kind of
special characters like { } in explanations confuses people, and they
have a tendency to think that they should type those special
characters.  A solution I wish we could use is to use special
formatting for the value of X in the text `ipfs get X`, to make it
really clear it's mean as a variable.  Failing that, I couldn't think
of anything clearer or less confusing that just using "X" in this
case.

People may have experiences doing this in a different/better way, in
which case, feel free to ignore this change.

mhucka committed:
Edit -- minor verbiage change + linkification

mhucka committed:
Copy of missing file, taken from ../replicate-with-control/

mhucka committed:
Changed image links to be relative

The links originally pointed to blobs on GitHub, but didn't resolve to
images in my local environment for some reason. Changing them to
relative links seems to make sense for the eventually installation
anyway, so I went ahead and did that, to feed two birds with one scone.

ebarry committed:
Merge pull request #23 from datatogether/mike-minor-changes

Mike minor changes - thanks Mike for capitalizing IPFS throughout, as well as updating image links!!! i was wondering about that.

b5 commented on browse-datasets/README.md:
```diff
@@ -1,32 +1,41 @@
+* Return to your web browser, and signup/signin to http://archivers.co/
 
-### Step 2.
+* Navigate to the content page for `CAFO_Density.zip`: http://archivers.co/content/12208b35b5c819ef6a8bb5df4657acbed1fbf084f8efe3174b3896db0058d91e82e5
```
Can we use this url instead? http://archivers.co/content/QmQU7RoBd8FWAkbePif7Xw5cjoL1Wb6wxCefdTuoA7XGa7

mhucka replied:
OK, done in commit 4b0e9eb

mhucka committed:
Fix hash, which needs to be base58-encoded

dcwalk reviewed (approved):
Great! Whoever merges will need to rebuild locally and push to gh-pages with the bash script!

mhucka reviewed (approved):
This is great work by @ebarry and really moved the ball forward on this tutorial.

mhucka commented:
@ebarry I'm not deleting branch liz-test-branch in case you want to keep it around, but if you don't, can you go ahead and delete it?
